opm/output/eclipse/InteHEAD.cpp

```diff
@@ -702,6 +703,8 @@ Opm::RestartIO::InteHEAD::variousParam(const int version,
     // ih_200: Usage unknown, value fixed across reference cases.
     this->data_[ih_200] = 1;
 
+    // NSTRA2  = 240
+    this->data_[NSTRA2] = 1 + (num_tracer_comps > 0) + num_tracer_comps;
```

Comment:
This changes INTEHEAD in every run since we now output a value that's at least 1, regardless of the number of tracers, where we previously would output zero. Is that intentional?